app/questionnaire/path_finder.py

```diff
@@ -92,9 +114,12 @@ def _block_index_for_block_id(blocks, block_id):
             None,
         )
 
-    def _build_path(self, blocks, path):
+    def _build_path(self, blocks, path, current_location):
         # Keep going unless we've hit the last block
         block_index = 0
+        for_list = self.schema.get_repeating_list_for_section(
```

Comment:
As above, just might make line 137 and 141 a bit easier out of context

Reply:
https://github.com/ONSdigital/eq-survey-runner/pull/2248#discussion_r313362102